src/css/paginator.styl

```diff
@@ -0,0 +1,25 @@
+.paginator
+  font-size: .86em
+  line-height: 1.37em
+  text-align: center
+
+  .paginator-button
+    @extends $reset-button
+    margin: 5px
+
+    &:hover
```

Comment:
Add also `&:focus`. Generally hover and focus styles usually should be the same and if one is defined, the other also should be for accessibility and consistency.